### PR TITLE
domain: print some logs to investigate the leak

### PR DESCRIPTION
**Pull request by wjhuang2016**
### What problem does this PR solve?

Problem Summary:

Just for the investigation, these codes will be removed later.

### What is changed and how it works?

What's Changed:
Print logs.

### Check List 

Tests 

- No code

Side effects

### Release note 
- No Release note.

### CODE REVIEW

File: domain/domain_test.go [hunk 1]
```diff
@@ -133,19 +135,91 @@ func interestingGoroutines() (gs []string) {
...
 		t.Logf("TestInfo: BeforeTest %s", str)
 	}
-	testleak.BeforeTest()
+	BeforeTest()
 	defer func() {
 		for _, str := range interestingGoroutines() {
 			t.Logf("TestInfo: AfterTest %s", str)
 		}
-		testleak.AfterTestT(t)()
+		AfterTestT(t)()
```
Comment (zimulala) [inline domain/domain_test.go:222]: Could we put it to line219？

File: domain/domain_test.go [hunk 2]
```diff
@@ -133,19 +135,91 @@ func interestingGoroutines() (gs []string) {
...
+		}()
+
+		var leaked []string
+		for i := 0; i < cnt; i++ {
+			leaked = leaked[:0]
+			for _, g := range interestingGoroutines() {
+				if !beforeTestGoroutines[g] {
+					leaked = append(leaked, g)
+					for str := range beforeTestGoroutines {
+						logutil.BgLogger().Info("checkLeakAfterTest", zap.String("L180", str))
```
Comment (zimulala) [inline domain/domain_test.go:180]: Maybe we can add`i` to log, then it is more readable.

File: domain/domain_test.go [hunk 3]
```diff
@@ -133,19 +135,91 @@ func interestingGoroutines() (gs []string) {
 	return
 }
```
Comment (zimulala) [inline domain/domain_test.go:137]: Could we add a "TODO" here?

Comment (wjhuang2016) [reply domain/domain_test.go:137]: // For debug only, will be removed later. In L95, 
I think it's enough to remind us.

Comment (zimulala) [reply domain/domain_test.go:138]: Got it.

**Pull request merged**